src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -1,27 +1,246 @@
+
+    /** Array of card objects */
+    cardList: paymentPropTypes.cardListPropTypes,
+
+    /** Wallet balance transfer props */
+    walletTransfer: paymentPropTypes.walletTransferPropTypes,
+
     ...withLocalizePropTypes,
 };
 
-const TransferBalancePage = props => (
-    <ScreenWrapper>
-        <KeyboardAvoidingView>
-            <HeaderWithCloseButton
-                title={props.translate('common.transferBalance')}
-                shouldShowBackButton
-                onBackButtonPress={() => Navigation.goBack()}
-                onCloseButtonPress={() => Navigation.dismissModal(true)}
-            />
-        </KeyboardAvoidingView>
-    </ScreenWrapper>
-);
+const defaultProps = {
+    userWallet: {},
+    bankAccountList: [],
+    cardList: [],
+    walletTransfer: {
+        transferAmount: 0,
+    },
+};
+
+class TransferBalancePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.paymentTypes = [
+            {
+                key: CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT,
+                title: this.props.translate('transferAmountPage.instant'),
+                description: this.props.translate('transferAmountPage.instantSummary', {
+                    amount: this.props.numberFormat(
+                        0.25,
+                        {style: 'currency', currency: 'USD'},
+                    ),
+                }),
+                icon: Expensicons.Bolt,
+                type: CONST.PAYMENT_METHODS.DEBIT_CARD,
+            },
+            {
+                key: CONST.WALLET.TRANSFER_METHOD_TYPE.ACH,
+                title: this.props.translate('transferAmountPage.ach'),
+                description: this.props.translate('transferAmountPage.achSummary'),
+                icon: Expensicons.Bank,
+                type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
+            },
+        ];
+
+        this.transferBalance = this.transferBalance.bind(this);
+
+        const selectedAccount = this.getSelectedAccount();
+        PaymentMethods.saveWalletTransferAmountAndAccount(
+            this.props.userWallet.currentBalance,
+            selectedAccount ? selectedAccount.id : '',
+        );
+    }
+
+    /**
+     * Get the selected/default Account for wallet tsransfer
+     * @returns {Object|undefined}
+     */
+    getSelectedAccount() {
+        const paymentMethods = PaymentUtils.getPaymentMethods(
+            this.props.bankAccountList,
+            this.props.cardList,
+        );
+
+        const defaultAccount = _.find(
+            paymentMethods,
+            method => method.id === lodashGet(this.props, 'userWallet.walletLinkedAccountID', ''),
+        );
+        const selectedAccount = this.props.walletTransfer.selectedAccountID
+            ? _.find(
+                paymentMethods,
+                method => method.id === this.props.walletTransfer.selectedAccountID,
+            )
+            : defaultAccount;
+
+        return selectedAccount;
+    }
```

Comment:
I don't see any selected account by default. `userWallet.walletLinkedAccountID` does not have any value.

Reply:
I _think_ we have to call the `SetWalletLinkedAccount` command to set one. @stitesExpensify would maybe know for sure.

Reply:
Yeah we have to set the account manually for now. We should start defaulting them in the future though..